docs/nb/Simple_Scripts.ipynb

```diff
@@ -184,6 +184,36 @@
   },
   {
    "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "----"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Meta Data :: specdb_meta\n",
+    "    specdb_meta -h\n",
+    "    usage: specdb_meta [-h] [--tol TOL] [-g GROUP] [--db_file DB_FILE] coord dbase\n",
+    "\n",
+    "    specdb_meta script v0.1\n",
+    "\n",
+    "    positional arguments:\n",
+    "      coord                 Coordinates, e.g. J081240.7+320809 or 122.223,-23.2322\n",
+    "                            or 07:45:00.47,34:17:31.1\n",
+    "      dbase                 Database [igmspec,uvqs,all,priv]\n",
+    "\n",
+    "    optional arguments:\n",
+    "      -h, --help            show this help message and exit\n",
+    "      --tol TOL             Maximum offset in arcsec [default=5.]\n",
+    "      -g GROUP, --group GROUP\n",
+    "                            Restrict on Group (e.g. BOSS_DR12)\n",
+    "      --db_file DB_FILE     Full path of db_file"
+   ]
```

Comment:
Can provide an example or two that work?  I tried the following and all failed:
`specdb_meta J081240.7+320809 igmspec`
`specdb_meta J081240.7+320809 all`
`specdb_meta 122.223,-23.2322 igmspec`

Error:
`Traceback (most recent call last):
  File "/Users/burchett/Applications/anaconda/bin/specdb_meta", line 6, in <module>
    exec(compile(open(__file__).read(), __file__, 'exec'))
  
File "/Users/burchett/Dropbox/Code/specdb/bin/specdb_meta", line 16, in <module>
    grab_meta.main(args)
  
File "/Users/burchett/Dropbox/Code/specdb/specdb/scripts/grab_meta.py", line 36, in main
    Specdb = load_db(args.dbase, db_file=args.db_file, **kwargs)
  
File "/Users/burchett/Dropbox/Code/specdb/specdb/utils.py", line 115, in load_db
    Specdb = IgmSpec(**kwargs)
  
File "/Users/burchett/Dropbox/Code/specdb/specdb/specdb.py", line 462, in __init__
    SpecDB.__init__(self, db_file=db_file, skip_test=skip_test, **kwargs)
  
File "/Users/burchett/Dropbox/Code/specdb/specdb/specdb.py", line 50, in __init__
    self.qcat = QueryCatalog(self.hdf, verbose=self.verbose, **kwargs)
  
File "/Users/burchett/Dropbox/Code/specdb/specdb/query_catalog.py", line 50, in __init__
    self.load_cat(hdf)
  
File "/Users/burchett/Dropbox/Code/specdb/specdb/query_catalog.py", line 82, in load_cat
    self.group_dict = json.loads(spdbu.hdf_decode(hdf['catalog'].attrs['GROUP_DICT']))
  
File "h5py/_objects.pyx", line 54, in h5py._objects.with_phil.wrapper (/Users/ilan/minonda/conda-bld/work/h5py/_objects.c:2579)
  
File "h5py/_objects.pyx", line 55, in h5py._objects.with_phil.wrapper (/Users/ilan/minonda/conda-bld/work/h5py/_objects.c:2538)
  
File "/Users/burchett/Applications/anaconda/lib/python2.7/site-packages/h5py/_hl/attrs.py", line 52, in __getitem__
    attr = h5a.open(self._id, self._e(name))
  
File "h5py/_objects.pyx", line 54, in h5py._objects.with_phil.wrapper (/Users/ilan/minonda/conda-bld/work/h5py/_objects.c:2579)
  
File "h5py/_objects.pyx", line 55, in h5py._objects.with_phil.wrapper (/Users/ilan/minonda/conda-bld/work/h5py/_objects.c:2538)
  
File "h5py/h5a.pyx", line 77, in h5py.h5a.open (/Users/ilan/minonda/conda-bld/work/h5py/h5a.c:2088)
KeyError: "Can't open attribute (Can't locate attribute: 'group_dict')"`